src/css/_nav.scss

```diff
@@ -7,6 +7,11 @@ header {
   width: 100%;
   justify-content: space-between;
   align-items: center;
+  position: sticky; 
```

Comment:
Oh I've never used this before - ill have to google to see what it does. Nice work!